automation/src/github.py

```diff
@@ -0,0 +1,87 @@
+import re
+from collections import defaultdict, namedtuple
+from typing import Any, Generator, Literal
+
+import os
+import httpx
+from markdown_it import MarkdownIt
+from markdown_it.tree import SyntaxTreeNode
+
+Issue = namedtuple("Issue", "metadata body")
+
+class Episode:
+    def __init__(self, episode_id: int, issue_fields: list):
```

Comment:
```suggestion
    def __init__(self, episode_id: int, issue_fields: list) -> None:
```

Missing return type. (This will all get automatic feedback once I get mypy set up.)